lib/rpm_package.py

```diff
@@ -140,13 +140,17 @@ def _load(self):
         super(RPM_Package, self)._load()
         try:
             # load distro files
-            files = self.package_data.get('files').get(
-                self.distro.lsb_name).get(self.distro.version)
-
-            self.build_files = files.get('build_files', None)
-            if self.build_files:
-                self.build_files = os.path.join(
-                    self.package_dir, self.build_files)
+            files = self.package_data.get('files', {}).get(
+                self.distro.lsb_name, {}).get(self.distro.version, {})
```

Comment:
We agreed on naming the distro centOS everywhere. The Python class remains to be CentOS, but that is not a problem because there is a dict between distro names and corresponding Python classes.

I am sending patches to handle this. They will keep backwards compatibility, which I hope we can remove on OP Host OS 2.0.